core/src/main/java/de/leonkoth/blockparty/floor/Floor.java

```diff
@@ -28,7 +28,10 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.util.ArrayList;
+import java.util.Collections;
```

Comment:
```suggestion
```
Sorry, this is no longer necessary since the call to `Collections#shuffle` has been removed.

Reply:
Done